EzMap.Domain/Models/User.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace EzMap.Domain;
+
+public class User
+{
+    public int UserId { get; set; }
+    public string? DisplayName { get; set; }
+    public string? UserName { get; set; }
+    public string? Email { get; set; }
+    public string? Password { get; set; }
```

Comment:
Password nullable?

Reply:
you are right, gonna update this soon

Reply:
No need to reply, if gonna fix it then just fix it, if need to be discussed then reply